features/bevragen/zoek-fout-cases.feature

```diff
@@ -106,17 +106,39 @@ Rule: Er moet een valide zoek type worden opgegeven
     | status   | 406                                                         |
     | instance | /haalcentraal/api/brp/personen                              |
 
-  Scenario: */* als Accept contenttype wordt ondersteund
+  Abstract Scenario: '<accept media type>' als Accept contenttype wordt ondersteund
```

Comment:
deze feature heet "zoek-fout-cases", maar dit is geen foutcase. Is dat een handige plek om dit scenario in te stoppen?